ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -45,6 +45,7 @@ type Service struct {
 	ecsContext        *context.ECSContext
 	deploymentConfig  *ecs.DeploymentConfiguration
 	loadBalancer      *ecs.LoadBalancer
```

Comment:
I guess we don't need this variable now.

Reply:
See above to why we might need this.